source/sixve.f90

```diff
@@ -211,397 +211,365 @@ subroutine envarsv(dpsv,oidpsv,rvv,ekv)
+    if(abs(fokm) <= pieni) goto 20
+    if(kz1 == 2) then
+      ih1 = 1
+      ih2 = 2
     else
 !  RECTANGULAR MAGNET VERTICAL
-      ih1=2
-      ih2=1
-    endif
-    do 50 j=1,napx
-      fok(j)=fokm/dpsq(j)
-      rho(j)=(one/ed(l))*dpsq(j)
-      fok1(j)=(tan_mb(fok(j)*half))/rho(j)
-      si(j)=sin_mb(fok(j))
-      co(j)=cos_mb(fok(j))
-      al(1,ih1,j,l)=one
-      al(2,ih1,j,l)=rho(j)*si(j)
-      al(3,ih1,j,l)=zero
-      al(4,ih1,j,l)=one
-  al(5,ih1,j,l)=((-one*dpsv(j))*((rho(j)*(one-co(j)))/dpsq(j)))*c1e3 !hr06
-  al(6,ih1,j,l)=((-one*dpsv(j))*((two*tan_mb(fok(j)*half))/dpsq(j)))&!hr06
-  &*c1e3                                                              !hr06
-      sm1(j)=cos_mb(fok(j))
-      sm2(j)=sin_mb(fok(j))*rho(j)
-      sm3(j)=(-one*sin_mb(fok(j)))/rho(j)                            !hr06
-      sm12(j)=el(l)-sm1(j)*sm2(j)
-      sm23(j)=sm2(j)*sm3(j)
-      as3(j)=(-one*rvv(j))*(((dpsv(j)*rho(j))/(two*dpsq(j)))*sm23(j)-&!hr06
-  &(rho(j)*dpsq(j))*(one-sm1(j)))                                     !hr06
-      as4(j)=((-one*rvv(j))*sm23(j))/c2e3                            !hr06
-      as6(j)=((-one*rvv(j))*(el(l)+sm1(j)*sm2(j)))/c4e3              !hr06
-  as(1,ih1,j,l)=(el(l)*(one-rvv(j))-rvv(j)*((dpsv(j)**2/            &!hr06
-  &(four*dpd(j)))*sm12(j)+dpsv(j)*(el(l)-sm2(j))))*c1e3               !hr06
-  as(2,ih1,j,l)=(-one*rvv(j))*((dpsv(j)/((two*rho(j))*dpsq(j)))*    &!hr06
-  &sm12(j)-(sm2(j)*dpsq(j))/rho(j))+fok1(j)*as3(j)                    !hr06
-      as(3,ih1,j,l)=as3(j)
-      as(4,ih1,j,l)=as4(j)+(two*as6(j))*fok1(j)                      !hr06
-      as(5,ih1,j,l)=(as6(j)*fok1(j)**2                              &!hr06
-  &-(rvv(j)*sm12(j))/(c4e3*rho(j)**2))+fok1(j)*as4(j)                 !hr06
-      as(6,ih1,j,l)=as6(j)
+      ih1 = 2
+      ih2 = 1
+    end if
+    do j=1,napx
+      fok  = fokm/dpsq(j)
+      rho  = (one/ed(l))*dpsq(j)
+      fok1 = (tan_mb(fok*half))/rho
+      si   = sin_mb(fok)
+      co   = cos_mb(fok)
+      al(1,ih1,j,l) = one
+      al(2,ih1,j,l) = rho*si
+      al(3,ih1,j,l) = zero
+      al(4,ih1,j,l) = one
+      al(5,ih1,j,l) = ((-one*dpsv(j))*((rho*(one-co))/dpsq(j)))*c1e3
+      al(6,ih1,j,l) = ((-one*dpsv(j))*((two*tan_mb(fok*half))/dpsq(j)))*c1e3
+
+      sm1  = cos_mb(fok)
+      sm2  = sin_mb(fok)*rho
+      sm3  = (-one*sin_mb(fok))/rho
+      sm12 = el(l)-sm1*sm2
+      sm23 = sm2*sm3
+      as3  = (-one*rvv(j))*(((dpsv(j)*rho)/(two*dpsq(j)))*sm23-(rho*dpsq(j))*(one-sm1))
+      as4  = ((-one*rvv(j))*sm23)/c2e3
+      as6  = ((-one*rvv(j))*(el(l)+sm1*sm2))/c4e3
+      as(1,ih1,j,l) = (el(l)*(one-rvv(j))-rvv(j)*((dpsv(j)**2/(four*dpd(j)))*sm12+dpsv(j)*(el(l)-sm2)))*c1e3
+      as(2,ih1,j,l) = (-one*rvv(j))*((dpsv(j)/((two*rho)*dpsq(j)))*sm12-(sm2*dpsq(j))/rho)+fok1*as3
+      as(3,ih1,j,l) = as3
+      as(4,ih1,j,l) = as4+(two*as6)*fok1
+      as(5,ih1,j,l) = (as6*fok1**2-(rvv(j)*sm12)/(c4e3*rho**2))+fok1*as4
+      as(6,ih1,j,l) = as6
 !--VERTIKAL
```

Comment:
Degermanizing required :)

Reply:
I de-germanised and norwegianised it. True story.